crates/common_utils/src/pii.rs

```diff
@@ -88,20 +98,103 @@ where
 }
 
 /// Email address
-#[derive(Debug)]
-pub struct Email;
+#[derive(serde::Serialize, serde::Deserialize, Debug, Clone, PartialEq, Eq, Default)]
+pub struct Email(
+    Secret<String>
+);
+
+impl From<String> for Email {
+    fn from(value: String) -> Self {
+        Email(Secret::new(value))
+    }
+}
```

Comment:
We'd want to avoid this, as this doesn't validate if an email is correct. Construction of an email from a `String` or `&str` could fail, so the only way to perform that conversion is using the `FromStr` trait.